mealprep/server.js

```diff
@@ -0,0 +1,51 @@
+const dotenv = require("dotenv");
```

Comment:
Is there a reason these files are .js and not .jsx?

Reply:
I'll switch it to jsx, just wasn't working as jsx for some reason and wanted to get it working for demo, now that I have more time should be no prob

Reply:
server.js will stay as .js. i think